api/pkg/service/resource/resource.go

```diff
@@ -243,7 +243,7 @@ func initResource(r model.Resource) *resource.Resource {
 		MinPipelinesVersion: lv.MinPipelinesVersion,
 		WebURL:              lv.URL,
 		RawURL:              replaceGHtoRaw.Replace(lv.URL),
-		UpdatedAt:           lv.UpdatedAt.UTC().String(),
+		UpdatedAt:           lv.ModifiedAt.UTC().String(),
```

Comment:
I think we should have a test that validates that the modified time is what is returned.